more/free-programming-cheatsheets.md

```diff
@@ -101,6 +102,11 @@
 * [jQuery CheatSheet](https://htmlcheatsheet.com/jquery/) (HTML)
 
 
+#### React.js
+
+* [React Cheatsheet](https://www.codecademy.com/learn/react-101/modules/react-101-jsx-u/cheatsheet) - Codecademy (React)
```

Comment:
Tag use to mean the format or platform where the resource is writed, not what kind of resource is

```markdown
* [React Cheatsheet](https://www.codecademy.com/learn/react-101/modules/react-101-jsx-u/cheatsheet) - Codecademy (HTML)
```